R/clean_names.R

```diff
@@ -65,13 +66,13 @@
 #' x %>%
 #'   clean_names(case = "upper_camel", abbreviations = c("ID", "DOB"))
 #'
-clean_names <- function(dat, ...) {
+clean_names <- function(dat, ..., set_labels = FALSE) {
```

Comment:
Please make `set_labels` an argument to the methods where it applies rather than the generic.  There may be some scenarios where `set_labels` does not make sense.

Reply:
Done